src/navs/index.js

```diff
@@ -2230,10 +2230,6 @@ export const docsMenu = {
                         {
                             name: 'Max AI',
                             url: '/docs/max-ai',
-                            badge: {
-                                title: 'Beta',
-                                className: 'uppercase !bg-blue/10 !text-blue !dark:text-white !dark:bg-blue/50',
-                            },
                         },
```

Comment:
Now that there's a full product page, `/docs/max-ai` seems somewhat duplicated – though actually it goes into a _different_ kind of detail into the feature. What do you think?

Reply:
I defer to @ivanagas here, but I'd say we should merge them